testapps/test_elasticsearch/tests/test_prod_es_indices.py

```diff
@@ -55,7 +55,7 @@ def test_prod_config(self):
     },
     {
         "alias": "hqusers",
-        "index": "test_hqusers_2017-02-03",
+        "index": "test_hqusers_2017-03-22",
```

Comment:
why do users have to be reindexed? if the user model changes in the form reindexer then it will automatically be queued to be transformed to the elasticsearch